platform/pulpcore/app/models/repository.py

```diff
@@ -97,23 +97,24 @@ class Importer(ContentAdaptor):
     Fields:
 
         feed_url (models.TextField): The URL of an external content source.
-        validate (models.BooleanField): Validate the imported context.
+        validate (models.BooleanField): If True, the plugin will validate imported files.
```

Comment:
s/imported/downloaded/  might be clearer.

Reply:
Is this also used for uploads?

Reply:
dunno

Reply:
@dkliban, I am going to leave this for today and merge. If uploads don't use this field, maybe we should specify that it is only for downloads.